#
# Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
#
upstream patch for
97884 - issue in symbols/lk
https://bugs.freedesktop.org/show_bug.cgi?id=97884
https://cgit.freedesktop.org/xkeyboard-config/commit/?id=e6f39df2c57c63f8af1d0ebb762f2f989fb4e512
--- a/symbols/lk	2015-05-26 22:02:47.000000000 +0200
+++ b/symbols/lk	2016-09-21 13:48:27.000000000 +0200
@@ -164,16 +164,16 @@
   key <AD11> { [   bracketleft,     braceleft,    VoidSymbol ] };
   key <AD12> { [  bracketright,    braceright,    VoidSymbol ] };
 
-  key <AE01> { [           one,        exclam,   dead_macron ] };
-  key <AE02> { [           two,            at, dead_belowdot ] };
-  key <AE03> { [         three,    numbersign, dead_abovedot ] };
-  key <AE04> { [          four,        dollar,      currency ] };
-  key <AE05> { [          five,       percent,           UB0 ] };
-  key <AE06> { [           six,   asciicircum,    onequarter ] };
-  key <AE07> { [         seven,     ampersand,       onehalf ] };
-  key <AE08> { [         eight,      asterisk, threequarters ] };
-  key <AE09> { [          nine,     parenleft,    VoidSymbol ] };
-  key <AE10> { [          zero,    parenright,    VoidSymbol ] };
+  key <AE01> { [             1,        exclam,   dead_macron ] };
+  key <AE02> { [             2,            at, dead_belowdot ] };
+  key <AE03> { [             3,    numbersign, dead_abovedot ] };
+  key <AE04> { [             4,        dollar,      currency ] };
+  key <AE05> { [             5,       percent,           UB0 ] };
+  key <AE06> { [             6,   asciicircum,    onequarter ] };
+  key <AE07> { [             7,     ampersand,       onehalf ] };
+  key <AE08> { [             8,      asterisk, threequarters ] };
+  key <AE09> { [             9,     parenleft,    VoidSymbol ] };
+  key <AE10> { [             0,    parenright,    VoidSymbol ] };
   key <AE11> { [         minus,    underscore,         U2014 ] };
   key <AE12> { [         equal,          plus,    VoidSymbol ] };
 
